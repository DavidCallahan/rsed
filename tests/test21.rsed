$text = 'double " ' "single ' "
print $text
print quote($text)
print quote("'", $text)
print quote("\"", $text)
