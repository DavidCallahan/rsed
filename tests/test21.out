double " single ' 
"double \" single ' "
'double " single \' '
"double \" single ' "
